pymatgen/analysis/tests/test_phase_diagram.py

```diff
@@ -90,30 +91,37 @@ def test_str(self):
     def test_read_csv(self):
         entries = EntrySet.from_csv(str(module_dir / "pdentries_test.csv"))
         self.assertEqual(entries.chemsys, {"Li", "Fe", "O"}, "Wrong elements!")
-        self.assertEqual(len(entries), 492, "Wrong number of entries!")
+        self.assertEqual(len(entries), 490, "Wrong number of entries!")
 
 
 class TransformedPDEntryTest(unittest.TestCase):
     def setUp(self):
+        # NOTE a 1:1 isn't the best test? maybe a tertiary system should be used.
```

Comment:
Agree, thanks for adding note

Reply:
shall I update the test to something more complicated?

Reply:
I have updated this test now to use three terminal compositions

Reply:
Thanks!